CONTRIBUTING.md

```diff
@@ -1,27 +1,88 @@
 
 ## Joining the community
 
-BIDS is a growing community of neuroimaging enthusiasts, and we want to make our resources accessible to and engaging for as many researchers as possible.
-We therefore require that all contributions adhere to our [Code of Conduct](http://www.brainhack.org/code-of-conduct.html).
+BIDS - the [Brain Imaging Data Structure][bids] - is a growing community of neuroimaging enthusiasts, and we want to make our resources accessible to and engaging for as many researchers as possible.
+
+We therefore require that all contributions **adhere to our [Code of Conduct](CODE_OF_CONDUCT.md)**.
+
+How do you know that you're a member of the BIDS community? You're here! You know that BIDS exists! You're officially a member of the community. It's THAT easy! Welcome! :joy::raised_hands:
+
+
+## Get in touch
+
+There are lots of ways to get in touch with the team maintaining the BIDS Starter Kit.
+
+* Our channel in the [BrainHack slack team][brainhack-slack-starterkit]
+  * Click [here][brainhack-slack-invite] for an invite to the slack workspace
+* Our [Gitter channel][gitter]
+* The [BIDS mailing list][bids-mailinglist]
+
+If you're here during summer 2018 :icecream::palm_tree:, you should reach out to our lovely [Google Summer of Code][gsoc] student [Patrick Park][patrick-github]. He'll be monitoring all the channels above and it would really help his project along if you said hello and passed along any feedback you have :purple_heart:.
+
+<img align="right" width="50%" src="https://linuxcentro.com.br/wp-content/uploads/2017/04/github-520x350.jpg" alt="Two github cats working together"/>
+
+## Contributing through GitHub
+
+[git][git] is a really useful tool for version control. [GitHub][github] sits on top of git and supports collaborative and distributed working.
+
+We know that it can be daunting to start using git and GitHub if you haven't worked with them in the past, but the BIDS Starter Kit maintainers are here to help you figure out any of the jargon or confusing instructions you encounter! :heart:
+
+In order to contribute via GitHub you'll need to set up a free account and sign in. Here are some [instructions](https://help.github.com/articles/signing-up-for-a-new-github-account/) to help you get going. Remember that you can ask us any questions you need to along the way.
+
+## Writing in markdown
+
+GitHub has a helpful page on [getting started with writing and formatting on GitHub](https://help.github.com/articles/getting-started-with-writing-and-formatting-on-github).
+
+Most of the writing that you'll do will be in [Markdown][markdown]. You can think of Markdown as a few little symbols around your text that will allow GitHub to render the text with a little bit of formatting. For example you could write words as **bold**, or in *italics*, or as a [link][rick-roll] to another webpage.
```

Comment:
Do you think it's helpful to have these examples (i.e., the **bold** text), since someone reading the guide wouldn't see how they were done unless they knew to look at the source code ? Which, if they haven't seen markdown before, seems like it would be overwhelming to find.

Reply:
+1 to this, so maybe something like **bold**  = \*\*bold** (so after the markdown resolves, readers can see the source characters as well too)

Reply:
The modified text looks great&mdash; there will be a non-zero percentage of folks who don't click-thru, so having this explicitly in the text will be helpful, I think !

Reply:
Edited in the modified text!